pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -132,33 +132,39 @@ class Build {
     The test jobs all follow the same name naming pattern that is defined in the openjdk-tests repository.
     E.g. Test_openjdk11_hs_sanity.system_ppc64_aix
     */
-    def determineTestJobName(testType) {
-
-        def variant
+    def getTestJobParams(testType) {
+        def jobParams = [:]
+        def (level, group) = testType.tokenize('.')
+        jobParams.put('LEVELS', level)
+        jobParams.put('GROUPS', group)
         def number = getJavaVersionNumber()
-
+        jobParams.put('JDK_VERSIONS', number)
+        def variant
         switch (buildConfig.VARIANT) {
             case "openj9": variant = "j9"; break
             case "corretto": variant = "corretto"; break
             case "dragonwell": variant = "dragonwell"; break;
             default: variant = "hs"
         }
-
+        jobParams.put('JDK_IMPL', variant)
         def arch = buildConfig.ARCHITECTURE
         if (arch == "x64") {
             arch = "x86-64"
         }
-
-        def os = buildConfig.TARGET_OS
-
-        def jobName = "Test_openjdk${number}_${variant}_${testType}_${arch}_${os}"
-
+        def arch_os = "${arch}_${buildConfig.TARGET_OS}"    
+        def jobName = "Test_openjdk${number}_${variant}_${testType}_${arch_os}"
         if (buildConfig.ADDITIONAL_FILE_NAME_TAG) {
             switch (buildConfig.ADDITIONAL_FILE_NAME_TAG) {
-                case ~/.*XL.*/: jobName += "_xl"; break
+                case ~/.*XL.*/: 
```

Comment:
And eventually with the move to mixedref builds we may not need this case at all, but for now we will still keep it.  :)